ecommerce/models.py

```diff
@@ -373,6 +373,9 @@ def fulfill(self, payment_data):
             keep_failed_enrollments=True,
         )
 
+        # send the receipt emails
+        queue_ecommerce_order_receipt(self)
```

Comment:
This is calling the task synchronously - the celery `@task` adds a `.delay()` method  on the task function to make an asynchronous call.